x/asset/priviledges/clawback/types.go

```diff
@@ -0,0 +1,10 @@
+package clawback
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type BankKeeper interface {
+	SendCoins(ctx sdk.Context, senderAddr, recipientAddr sdk.AccAddress, amt sdk.Coins) error
```

Comment:
Is `senderAddr` a `string` or `sdk.AccAddress`?  Or does it not matter?

Reply:
dito